TourGuide_Horde/01_12_Durotar.lua

```diff
@@ -93,8 +93,8 @@ T Carry Your Weight |QID|791| |T|
 
 C From The Wreckage.... |QID|825| |N|East of Razor Hill, off the coast (62,50).| |NODEBUG|
 C A Solvent Spirit |QID|818| |N|Along the beaches.|
-K Kill Quilboars and Scouts |QID|837| |QO|Razormane Quilboar slain: 4/4| |N|Do the first half, southwest of Razor Hill for Scouts and Quilboars (51.6, 48.6)|
-K Kill Quilboars and Scouts |QID|837| |QO|Razormane Scout slain: 4/4| |N|Do the first half, southwest of Razor Hill for Scouts and Quilboars (51.6, 48.6)|
+K Razormane Quilboar |QID|837| |QO|Razormane Quilboar slain: 4/4|
+K Razormane Scout |QID|837| |QO|Razormane Scout slain: 4/4|
```

Comment:
Couple things here. We need to keep the note, in some form, and here's why. Kill objectives don't query pfQuest. This is because pfQuest has no knowledge of mob locations. It knows about start NPCs and turnin NPCs. That's generally it. Thus, in a K or C objective, we implement notes to provide a small description, but more importantly, to provide a coordinate for the player to follow... at least most of the time.

In addition, you point out something that is missing. Currently, |QO| tags also require |Q| tags. You supply the quest title (since the title of the K objective can be anything) and then also the quest objective. However, the |QO| must match **exactly** what the quest log objective says. In this case, it should be correct. So, we need to add a |Q| tag here for the quest title.

Reply:
I only listed the Mob names as the title, not the Quest name itself. So it shouldn't need the |Q| tag, but I'll revert the change.